src/support/slack/commands/toggle-site-audit.js

```diff
@@ -43,51 +43,234 @@ export default (context) => {
     }
   };
 
-  const handleExecution = async (args, slackContext) => {
-    const { say } = slackContext;
-    const [enableAuditInput, baseURLInput, auditTypeInput] = args;
+  const processCSVContent = async (fileContent) => fileContent
```

Comment:
we use the `csv` library: https://www.npmjs.com/package/csv - handles edge cases and you don't have to do validation yourself